bottlecap

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
+
+usage() {
+	echo "usage:"
+	echo "bottlecap [--dev] [--runtime $runtime] [--build-dir $build_dir]  [--container $container] assemblerargs..."
+	echo "  --dev: rebuild and reinstall mantle, the build scripts, and the ostree-releng-scipts before running coreos-assembler"
+	echo "  --runtime: (auto) specify whether to use podman or docker. Defaults to podman if it is in \$PATH"
+	echo "  --build-dir: Where to put build artifacts, defaults to the current directory"
+	echo "  --container: which coreos-assembler container to use"
+	echo "  --help: print this help message"
+	exit 0
+}
+
+dev=0
+runtime="podman"
+which podman > /dev/null 2> /dev/null || runtime="docker"
+build_dir="$(pwd)"
+container="quay.io/cgwalters/coreos-assembler"
+
+while :
+do
+	case "$1" in
+		"--dev")
+	  		dev=1
+			shift
+			;;
+		"--runtime")
+			shift
+			runtime="$1"
+			shift
+			;;
+		"--build-dir")
+			shift
+			build_dir="$1"
+			shift
+			;;
+		"--container")
+			shift
+			container="$1"
+			shift
+			;;
+		"--help")
+			usage
+			;;
+		*) # Rest of the args are for COSA
```

Comment:
Optional: take an explicit `--` here too; this way one could e.g. run `coreos-assembler -- --help` and get the "inner help".

Reply:
fixed.